src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.js

```diff
@@ -54,12 +54,9 @@ function Panel({
 
   useEffect(() => {
     editor.editor.on("refresh", updateWidth);
-    dismissNag(Nag.FIRST_GUTTER_CLICK);
+    dismissNag(Nag.FIRST_BREAKPOINT_ADD);
```

Comment:
`FIRST_GUTTER_CLICK` and `FIRST_BREAKPOINT_ADD` were doing the same thing, so just collapsed it.